<launch>
  <node pkg="nodelet" type="nodelet" name="standalone_nodelet"  args="manager" output="screen"/>

  <node pkg="nodelet" type="nodelet" name="MyNodeletClass" args="load gr_pointcloud_filter/MyNodeletClass standalone_nodelet" output="screen">
  </node>
</launch>